client/my-sites/earn/memberships/style.scss

```diff
@@ -315,6 +315,34 @@
 .memberships__dialog-sections {
 	display: grid;
 	grid-template: "item" 1fr / 1fr;
+	@include breakpoint-deprecated( ">600px" ) {
+		min-width: 580px;
+	}
+	.form-setting-explanation {
+		font-size: $font-body-small;
+	}
+	.memberships__dialog-advanced-options {
```

Comment:
There's so much override that I kinda feel like we can follow-up with simple new component. But good for this pr!